nose/test_orbit.py

```diff
@@ -3554,3 +3554,20 @@ def check_integrate_t_asQuantity_warning(o,funcName):
         assert raisedWarning, "Orbit method %s wit unitless time after integrating with unitful time should have thrown a warning, but didn't" % funcName
     return None  
 
+def test_orbitint_pythonfallback():
+    # Check if a warning is raised when the potential has no C integrator
+    from galpy.orbit import Orbit
+    bp= BurkertPotentialNoC() # BurkertPotentialNoC is already imported at the top of test_orbit.py
+    orb= Orbit([1.,0.1,1.1,0.1,0.,1.])
```

Comment:
This looks great and is almost ready to merge, but it would be better to test all four new warnings.

I think you should be able to test all 4 new warnings by just looping through all of the possible orbits. That is, start a loop here with
```
for orb in [Orbit([1.,0.1,1.1,0.1,0.,1.]),Orbit([1.,0.1,1.1,0.1,0.]),Orbit([1.,0.1,1.1,1.]),Orbit([1.,0.1,1.1])]:
```
and then put everything in this function within the loop. Probably best practice to move the ``ts= `` and ``import warnings`` lines to before the loop.

Yes, ``BurkertPotentialNoC`` has no ``normalize`` keyword, but you can call
```
bp.normalize(1.)
```
after you initialize ``bp`` to do the normalization.

I can make these final changes as well if you allow me to edit the pull request (see https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/).